shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/TxBody.hs

```diff
@@ -1076,6 +1117,8 @@ instance
           _poolMD = maybeToStrictMaybe md
         }
 
+{-
```

Comment:
These are commented out. Are they unused? Can they be deleted?

Reply:
Those are remnants of the switch to set algebras, Haven't been used in a long time.